README.md

```diff
@@ -0,0 +1,54 @@
+[![Research software impact](http://depsy.org/api/package/pypi/khmer/badge.svg)](http://depsy.org/package/python/khmer)
+![Supported Python versions](https://img.shields.io/pypi/pyversions/khmer.svg)
```

Comment:
This currently renders as 2.7, 3.3, and 3.4, since that is what was in the PyPI metadata as of release 2.0. Once the new release is cut, this will be updated automatically.